docs/book/src/user/quick-start.md

```diff
@@ -366,6 +366,17 @@ export DO_B64ENCODED_CREDENTIALS="$(echo -n "${DIGITALOCEAN_ACCESS_TOKEN}" | bas
 clusterctl init --infrastructure digitalocean
 ```
 
+{{#/tab }}
+
+{{#tab Outscale}}
```

Comment:
Must be added to the tab list above as well

Reply:
Thanks you, it is done